packages/dx-react-grid/docs/reference/table-header-row.md

```diff
@@ -40,6 +41,7 @@ sortingEnabled | boolean | Specifies whether sorting by a column is enabled.
 sortingDirection? | 'asc' &#124; 'desc' | Specifies the associated column's sorting direction.
 onSort | (parameters: { direction?: 'asc' &#124; 'desc' &#124; null, keepOther?: boolean }) => void | An event that initiates changing the column sorting direction. Keeps the current sorting state if `keepOther` is set to true. Cancels sorting by the current column if `direction` is set to null.
 showGroupingControls | boolean | Specifies whether to display a button that groups data by column.
+sortingComponent | ComponentType&lt;[TableHeaderRow.SortingControlProps](#tableheaderrowsortingcontrolprops)&gt; | | A component that renders a sorting control.
```

Comment:
Why cell knew something about sort label component here? I think that TableHeaderRow plugin should create element using this component and pass it to children property.

If we do so, we should also deprecate sorting properties on the cell component.

Reply:
This case won't work https://codesandbox.io/s/o9nzp7nk29

BC?